spanner/cloud-client/src/main/java/com/example/spanner/LoadCsvExample.java

```diff
@@ -213,38 +253,35 @@ public static void main(String... args) throws Exception {
     String projectId = options.getProjectId();
     String instanceId = args[0];
     String databaseId = args[1];
-    String tableId = args[2];
+    String tableName = args[2];
     String filepath = args[3];
-    CommandLineParser clParser = new DefaultParser();
-    CommandLine cmd = clParser.parse(opt, args);
 
     try {
+      // Initialize connection to Cloud Spanner
       connection = DriverManager.getConnection(
           String.format(
               "jdbc:cloudspanner:/projects/%s/instances/%s/databases/%s",
               projectId, instanceId, databaseId));
 
-      parseTableTypes(tableId);
-      parseTableHeaders(tableId);
-
+      parseTableColumns(tableName);
```

Comment:
parseTableColumns() is put into a try-catch block, but it doesn't throw in failure cases, am I missing something here?

Reply:
`parseTableColumns` I believe throws an  SQLException if the SELECT query for information schema data fails. The try block also wraps the FileReader and the CSVParser which can also throw IOExceptions.